### PR TITLE
Updating with 1.6.2-cs5 notes

### PR DESCRIPTION
**Pull request by moxiegirl**
Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

**Comment by moxiegirl** [comment]
ping @docker/docs-owners ping @alvinr

File: docs/sources/docker-hub-enterprise/release-notes.md [hunk 1]
```diff
@@ -18,6 +18,70 @@ page_keywords: docker, documentation, about, technology, understanding, enterpri
 
 ## Commercialy Supported Docker Engine
 
+### CS Docker Engine 1.6.2-cs5
+
+For customers running Docker Engine on [supported versions of RedHat Enterprise
+Linux](https://www.docker.com/enterprise/support/) with [SELinux
+enabled](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/
+6/html/Security-Enhanced_Linux/sect-Security-Enhanced_Linux-Working_with_SELinux
```
Comment (thaJeztah) [inline docs/sources/docker-hub-enterprise/release-notes.md]: Looks like you wrapped the URL here; better check if that still works because of that

**Comment by alvinr** [comment]
lgtm

File: docs/sources/docker-hub-enterprise/release-notes.md (hunk 1 above)
Comment (moxiegirl) [inline docs/sources/docker-hub-enterprise/release-notes.md]: Yep they work.

**Comment by thaJeztah** [comment]
Thanks for confirming, LGTM

**Pull request merged**